ocs_ci/ocs/ui/views.py

```diff
@@ -24,6 +24,24 @@
     ),
     "skip_tour": ('button[data-test="tour-step-footer-secondary"]', By.CSS_SELECTOR),
 }
+
+ibm_cloud_managed = (
```

Comment:
Please consider as a recommendation >  to move version-related logic to execute in BaseUI. Thus we'll have views.py clean constants, as was designed, such as we have in constants.py, otherwise we'll have 2 diff modules which determine version-to-locators relation.